src/Libraries/Nop.Data/Migrations/UpgradeTo470/AddIndexesMigration.cs

```diff
@@ -143,6 +143,20 @@ public override void Up()
                 IfDatabase(databaseType).Create.Index("IX_ProductReview_CustomerId").OnTable(nameof(ProductReview))
                     .OnColumn(nameof(ProductReview.CustomerId)).Ascending()
                     .WithOptions().NonClustered();
+
+            if(!Schema.Table(nameof(Product)).Index("IX_Product_Name").Exists())
+                IfDatabase(databaseType).Create.Index("IX_Product_Name")
```

Comment:
There is no need to check the database type; these indexes must be added to all supported DBMSs

Reply:
@skoshelev Done